Sources/FoundationInternationalization/Formatting/Number/IntegerParseStrategy.swift

```diff
@@ -32,8 +32,6 @@ extension IntegerParseStrategy: ParseStrategy {
         let trimmedString = value._trimmingWhitespace()
         if let v = parser.parseAsInt(trimmedString) {
             return Format.FormatInput(v)
-        } else if let v = parser.parseAsDouble(trimmedString) {
-            return Format.FormatInput(clamping: Int64(v))
```

Comment:
This one I'm not so sure. We don't know if this code path is not currently being used even though I can't personally hit it. Should we just leave it as is for now?